source/AI.h

```diff
@@ -42,62 +42,76 @@ class System;
 
 
 
-// This class is responsible for controlling all the ships in the game,
-// including the player's "flagship" - which is usually controlled via the
-// keyboard but can also make use of the AI's autopilot for landing, aiming,
-// or hyperspace travel between systems. The AI also tracks which actions
-// ships have performed, to avoid having the same ship try to scan or board
-// the same target over and over.
+/// This class is responsible for controlling all the ships in the game,
+/// including the player's "flagship" - which is usually controlled via the
+/// keyboard but can also make use of the AI's autopilot for landing, aiming,
+/// or hyperspace travel between systems. The AI also tracks which actions
+/// ships have performed, to avoid having the same ship try to scan or board
+/// the same target over and over.
 class AI {
 public:
-	// Any object that can be a ship's target is in a list of this type:
+	/// Any object that can be a ship's target is in a list of this type:
+///
 template <class Type>
 	using List = std::list<std::shared_ptr<Type>>;
-	// Constructor, giving the AI access to the player and various object lists.
+	/// Constructor, giving the AI access to the player and various object lists.
+	///
 	AI(const PlayerInfo &player, const List<Ship> &ships,
 			const List<Minable> &minables, const List<Flotsam> &flotsam);
 
-	// Fleet commands from the player.
+	/// Fleet commands from the player.
+	///
 	void IssueFormationChange(PlayerInfo &player);
 	void IssueShipTarget(const std::shared_ptr<Ship> &target);
 	void IssueAsteroidTarget(const std::shared_ptr<Minable> &targetAsteroid);
 	void IssueMoveTarget(const Point &target, const System *moveToSystem);
```

Comment:
This comment relates to all four of these function declarations, not just one of them.
Will Doxygen handle that or will it just associate it with the first method and then give the other nothing?

Reply:
with the first, it is kinda difficult to have a script recognize something like that

Reply:
If Doxygen can't handle them, I don't think comments like this, introducing groups of methods, should be using the Doxygen syntax.

Reply:
well using a script i have no real way of knowing if the comment describes a single function or all following

Reply:
if you have a good method tell me, but I would rather not go through all comments to look for this case

Reply:
I don't know of a better option, but I agree that this shouldn't be in doxygen.

Reply:
so apparently you can do groups in doxygen with:
///@{
void funca();
void funcb();
///@}

but we still have to go through each file

Reply:
Proposal when we know what we want to do about this, we make a checklist of files and whenever someone feels like it, they can go through a few, suggest changes and then we mark them of the list.
Or my preferred option: Just don't care about it, but that is up to you.